test/functional/Java16andUp/src/org/openj9/test/foreignMemoryAccess/TestCloseScope0.java

```diff
@@ -83,12 +83,25 @@
 	@Test(expectedExceptions=java.lang.IllegalStateException.class)
 	public static void closeScopeDuringAccess() throws Throwable {
 		/* Reflect setup */
-		Class c = Class.forName("jdk.internal.foreign.MemoryScope");
-		Method createShared = c.getDeclaredMethod("createShared", new Class[] {Object.class, Runnable.class, Cleaner.class});
-		createShared.setAccessible(true);
-		Method close = c.getDeclaredMethod("close");
+		Class MemoryOrResourceScope;
+		Method createShared;
+		Method close;
+		Object scope;
+
+		String version = System.getProperty("java.vm.specification.version");
+		if (Integer.parseInt(version) == 16) {
```

Comment:
We no longer support JDK16 so your changes only need to work with JDK17+. @EricYangIBM Can you update the PR accordingly and let me know?

Reply:
Moved test from Java16andUp to Java17andUp